src/modules/systemlib/airspeed.h

```diff
@@ -48,6 +47,29 @@
 
 __BEGIN_DECLS
 
+enum AIRSPEED_SENSOR_MODEL {
+	AIRSPEED_SENSOR_MODEL_MEMBRANE = 0,
+	AIRSPEED_SENSOR_MODEL_SDP3X,
```

Comment:
There also seems to be a desire for supporting multiple airspeed sensors (even for only this kind of testing). Getting some kind identifier in place for extending the calibration is the first step. I suppose we'd still need a completely separate dev id and model enum though.